lisa/tools/hyperv.py

```diff
@@ -91,6 +103,21 @@ def create_vm(
                 force_run=True,
             )
 
+        # configure COM ports if specified
+        if com_ports is None:
```

Comment:
It doesn't need to specify an empty com_ports, if it's not set.
```suggestion
        if com_ports:
            ...
```

Reply:
Changed.